src/main/java/com/alexlasota/medical_clinic_proxy/client/MedicalClinicClient.java

```diff
@@ -0,0 +1,28 @@
+package com.alexlasota.medical_clinic_proxy.client;
+
+import com.alexlasota.medical_clinic_proxy.configuration.FeignClientConfig;
+import com.alexlasota.medical_clinic_proxy.model.VisitDto;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@FeignClient(name = "medical-clinic", url = "${MEDICAL_CLINIC_URL:http://medical-clinic:8080}"
```

Comment:
Zweryfikuj czy odpalając aplikacje normalnie nie w kontenerze rowniez to dziala.
Moim zdaniem defaultem powinno byc jesli juz to localhost

Reply:
![Screenshot 2024-07-17 at 12 18 26](https://github.com/user-attachments/assets/1150edc8-6113-47ea-85ca-4d305bcc0212)